usd_for_web_demos/test.html

```diff
@@ -19,7 +19,7 @@
     // Load the usd file
     let loadUSDPromise = new Promise( (resolve, reject) => {
       let req = new XMLHttpRequest();
-      req.open("GET", "/" + filename, true);
+      req.open("GET", filename, true);

```

Comment:
This change is also part of #4

Reply:
Note that this requires changing the links in index.html to include this slash (which is part of this PR).